### DISCUSSION TITLE
Any update does not work anymore.

### DISCUSSION THREAD

**Discussion by gwenfisch** [community/community#10691] [Projects and Issues]
Hi,

Since just now, everything was working correctly.

But now, as soon as we want to modify a status, add a new issue or whatever the data is saved but nor persistent.
And then it is back to the previous state as if we did not modify or add anything.

We can't use projects at all anymore....

**Comment by Galadirith** [comment]
I also am seeing issue state (open or closed) failing to update. It seems like whatever model is responsible for the Projects beta view is not able to accept any updates from any of the sources it depends on at the moment.

**Comment by gwenfisch** [comment]
OK, now it seems resolved on my side. 

And you ?

**Reply by Galadirith** [reply]
Yeh I agree this does now appear to have resolved itself. There didn't appear to be any indication on https://www.githubstatus.com/ that anything had happened, but I appreciate given we are still in Beta there can be no guarantee about service right now, and no reason it should be tied into any public status system yet.
